stock_orderpoint_automatic_creation/models/product_category.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Daniel Campos <danielcampos@avanzosc.es> - Avanzosc S.L.
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = 'product.category'
+
+    create_orderpoints = fields.Selection(
```

Comment:
Original author decide that when user select "No" option in product or categories, then orderpoint is not created even if company settings are configured to create them.

if user select not to create orderpoint in product:
https://github.com/OCA/stock-logistics-warehouse/pull/308/files#diff-0743ce7eb081903f7439f511d86f3439R28
or in category:
https://github.com/OCA/stock-logistics-warehouse/pull/308/files#diff-0743ce7eb081903f7439f511d86f3439R30
then orderpoint is not created.

This allows user to avoid create orderpoint always for all products and categories if company are configured to do it.